stripe/_http_client.py

```diff
@@ -533,16 +607,32 @@ def __init__(
         assert urlfetch is not None
         self.urlfetch = urlfetch
 
-    def request(self, method, url, headers, post_data=None):
+    def request(
+        self, method, url, headers, post_data=None
```

Comment:
nit: more types for arguments?

Reply:
Done in https://github.com/stripe/stripe-python/pull/1232/commits/35322442b9b0bec00137ff48e11db29f96b79868